editor/utils/heartbeat.js

```diff
@@ -0,0 +1,50 @@
+export function setupHearthbeat() {
```

Comment:
`setupHearthbeat` is misspelled, should be `setupHeartbeat`

Reply:
thanks for reviewing @travislopes, I fixed this!